xml/bp_troubleshooting_cephadm.xml

```diff
@@ -118,8 +118,60 @@ systemctl status "ceph-$(cephadm shell ceph fsid)@<replaceable>service name</rep
   done
 </screen>
  </sect1>
- <sect1 xml:id="bp-troubleshooting-cephadm-container-images">
-   <title>List all Downloaded Container Images</title>
+ <sect1 xml:id="bp-troubleshooting-cephadm-configured-container-images">
+   <title>List Configured Container Images</title>
+   <para>
+     &cephadm; has a set of container images configured by default.
+   </para>
+   <para>
+     To get the value of the configured default container:
+   </para>
+<screen>&prompt.cephuser;ceph config get mgr mgr/cephadm/<replaceable>OPTION_NAME</replaceable></screen>
+   <para>
+     Where <replaceable>OPTION_NAME</replaceable> is any of the following names:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+       container_image_prometheus
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+       container_image_prometheus
+     </para>
```

Comment:
This seems to duplicate the previous item.

Reply:
oops, fixed!